docs/source/benchmark_utils.rst

```diff
@@ -10,3 +10,15 @@ Benchmark Utils - torch.utils.benchmark
 .. autoclass:: Timer
     :members:
     :show-inheritance:
+
+.. autoclass:: Measurement
+    :members:
+    :show-inheritance:
```

Comment:
does it make sense to show inheritance for classes inheriting from `object`?

Reply:
No, I suppose not.